cmake/testing.cmake

```diff
@@ -267,6 +267,21 @@ if(F3D_MODULE_OCCT)
   f3d_test(TestIGES spacer.igs "300,300")
 endif()
 
+if(F3D_MODULE_ASSIMP)
+  f3d_test(TestOFF teapot.off "300,300" "--up=+Z")
+  f3d_test(TestDXF PinkEggFromLW.dxf "300,300" "--bg-color=1,1,1 -p")
+
+  if(VTK_VERSION VERSION_GREATER 9.0.20210728)
+    f3d_test(TestFBX RED_BRICK_PILE_MEDUIM_5K.fbx "300,300")
+    f3d_test(TestDAE duck.dae "300,300")
+  endif()
+
+  # animations are broken with Assimp 5.1
+  if("${F3D_ASSIMP_VERSION}" VERSION_LESS "5.1")
```

Comment:
test equality with 5.0 instead ? without reading the CMakeLists this is not very clear